### PR TITLE
Added cvar for smuggle bonus

### PR DESCRIPTION
**Pull request by Ashnal**
Also subjectively cleaned it up a little bit

### CODE REVIEW

File: xItemSrc/xItemLib_v112.lua
Comment (minenice55) [inline xItemSrc/xItemLib_v112.lua:158]: would it be smart to use xitem ids here instead of hardcoded numbers?

**Review by minenice55 — COMMENTED** [review]
changes look good but I'll let you read my suggestion and implement it if you wish to (I'll merge right away if you don't)

File: xItemSrc/xItemLib_v112.lua
Comment (JugadorXEI) [reply xItemSrc/xItemLib_v112.lua:158]: You probably wanna have further cleanup in a different PR, I'd rather have this merged now for the cvar.

Comment (minenice55) [reply xItemSrc/xItemLib_v112.lua:158]: sounds good, merging...

**Review by minenice55 — APPROVED** [review]

**Pull request merged**
